src/pages/productListingPage/productListingPage.jsx

```diff
@@ -0,0 +1,42 @@
+import { Navbar, Card, Sidebar } from "../../components";
+import "./product-listing-page.css";
+
+export default function ProductListingPage() {
+  return (
+    <div className="page-wrap">
+      <section className="page-nav">
+        <Navbar></Navbar>
+      </section>
+
+      <section className="page-sidebar">
+        <Sidebar></Sidebar>
+      </section>
+
+      <section className="page-main">
+        <main className="main-content dui-util-spc-pad-2_4rem-m">
+          {[1,2,3,4,5,6,7,8].map((itemDetails) => {
+            // console.log(a);
```

Comment:
remove commented code.

Reply:
will be removed in upcoming refactoring.